src/app/components/Item/ItemTableRow.jsx

```diff
@@ -48,11 +50,14 @@ class ItemTableRow extends React.Component {
       bibId,
       searchKeywords,
     } = this.props;
-    const { closedLocations } = appConfig;
+    const { closedLocations, recapClosedLocations, nonRecapClosedLocations } = appConfig;
+    // const isRecap = LibraryItem.mapItem(item).isRecap;
+    const isRecap = item.isRecap;
+    const allClosed = closedLocations.concat((isRecap ? recapClosedLocations : nonRecapClosedLocations)).includes('');
```

Comment:
Why not just keep `all` as the value for `closedLocations`?

Reply:
Using `''` instead  of `all` is useful for filtering locations on the `HoldRequest` page, since the code filters out any location that begins with a string in the `closedLocations`, so `''` will filter everything out. At the moment it is maybe not strictly necessary since we are specifically checking for `allClosed` but I think it is a useful failsafe.